contracts/EthlanceBountyIssuer.sol

```diff
@@ -0,0 +1,95 @@
+
+  /**
+      This function is for inviting more arbiters, in case nobody
+      accepted in the first initial of invites.
+  */
+  function inviteArbiters(address[] memory arbiters, uint fee, uint bountyId) public {
+    // NOTE : shoulud we check that bountyId exists in StandardBounties ?
+    for(uint i = 0; i < arbiters.length; i ++){
+      arbitersBounties[arbiters[i]][bountyId]=ArbiterBounty(fee, false);
+    }
+  }
+
+  /**
+     This function creates a bounty in StandardBouties contract,
+     passing as issuers addresses of this contract and sender's
+     address. Also it stores addresses of invited arbiters (approvers)
+     and arbiter's fee for created bounty.
+  */
+  function issueAndContribute(string memory bountyData, uint deadline, address token, uint tokenVersion, uint depositAmount) public payable{
+    address[] memory arbiters=new address [](0);
+
+    // EthlanceBountyIssuer is the issuer of all bounties
+    address payable[] memory issuers=new address payable[](1);
+
+    address payable thisPayable=address(uint160(address(this)));
+    issuers[0]=thisPayable;
+
+    require(depositAmount > 0); // Contributions of 0 tokens or token ID 0 should fail
+
+    if (tokenVersion == 0){
+      require(msg.value == depositAmount);
+    } else if (tokenVersion == 20){
+
+      require(msg.value == 0); // Ensures users don't accidentally send ETH alongside a token contribution, locking up funds
+
+      require(IERC20(token).transferFrom(msg.sender,
+                                         address(this),
+                                         depositAmount));
+    } else if (tokenVersion == 721){
+      require(msg.value == 0); // Ensures users don't accidentally send ETH alongside a token contribution, locking up funds
+      IERC721(token).transferFrom(msg.sender,
+                                  address(this),
+                                  depositAmount);
+    } else {
+      revert();
+    }
+
+
+    // Also pass whatever value was sent to us forward
+    uint bountyId = standardBounties.issueAndContribute.value(msg.value)(thisPayable,
+                                                                         issuers,
+                                                                         arbiters,
+                                                                         bountyData,
+                                                                         deadline,
+                                                                         token,
+                                                                         tokenVersion,
+                                                                         depositAmount);
+
+  }
+
+  /**
+     Arbiter runs this function to accept invitation. If he's first, it'll
+     transfer fee to him and it'll add him as arbiter for the bounty.
+  */
+  function acceptArbiterInvitation(uint bountyId) public {
```

Comment:
Isn't it missing transfer of the fee to an arbiter, in this function?

Reply:
I wasn't sure the fee transfer was at this stage. Do arbiters get paid as soon as they accept the invitation?